build_daemon/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 dependencies:
   built_collection: ^5.0.0
   built_value: ^8.1.0
+  crypto: ^3.0.3
```

Comment:
Is this a new transitive dep when you exclude dev dependencies? Consider mentioning either way in the commit message.

Reply:
It is new  yes, I can mention it.